closure/testing/test/BUILD

```diff
@@ -31,28 +31,31 @@ closure_js_library(
     deps = ["//closure/library"],
 )
 
-closure_js_test(
-    name = "arithmetic_module_test",
-    srcs = ["arithmetic_module_test.js"],
-    defs = ["--entry_point=goog:arithmetic_module_test"],
-    language = "ECMASCRIPT6_STRICT",
-    deps = [
-        ":arithmetic_module_lib",
-        "//closure/library",
-        "//closure/library:testing",
-    ],
-)
-
-closure_js_test(
-    name = "arithmetic_scope_test",
-    srcs = ["arithmetic_scope_test.js"],
-    language = "ECMASCRIPT6_STRICT",
-    deps = [
-        ":arithmetic_module_lib",
-        "//closure/library",
-        "//closure/library:testing",
-    ],
-)
+# TODO(hochhaus): Enable test when upstream local file loader fix is included
```

Comment:
I'm looking into this issue further and I think my explanation is wrong. More soon...

Please hold off on looking more so that you don't waste your time.